packages/app/src/pages/Runner.vue

```diff
@@ -1,25 +1,37 @@
 <template>
   <SpecRunnerContainer
-    v-if="query.data.value?.currentProject"
-    :gql="query.data.value.currentProject"
+    v-if="queryResult"
+    :gql="queryResult"
   />
 </template>
 
 <script lang="ts" setup>
 import { gql, useQuery } from '@urql/vue'
-import { SpecPageContainerDocument } from '../generated/graphql'
+import { computed } from 'vue'
+import { SpecPageContainerDocument, SpecPageContainerQuery, SpecRunnerFragment } from '../generated/graphql'
 import SpecRunnerContainer from '../runner/SpecRunnerContainer.vue'
+import type { NonNullable } from '@packages/types'
 
 gql`
 query SpecPageContainer {
-  currentProject {
-    id
-    ...SpecRunner
-  }
+  ...SpecRunner
 }
 `
 
 const query = useQuery({ query: SpecPageContainerDocument })
+
+export type GqlWithCurrentProject = SpecPageContainerQuery & {
```

Comment:
This feels hacky, @tgriesser can we sync on this? Also notified in [slack](https://cypressio.slack.com/archives/C027U3ZEJ6Q/p1637307934004600)